chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -45,6 +45,11 @@ ORIGINAL_INSTANCES=$UPGRADEDIR/original_instances
 # List active instances on host before the live-migration starts
 openstack server list --insecure --all-projects --status active --host $host -f value -c ID > $ORIGINAL_INSTANCES
 
+while openstack --insecure compute service list --service nova-compute -f value -c State | grep down ; do
```

Comment:
Does it make sense to wait indefinitely?

Reply:
Yeah, if there is some other problem we don't want the upgrade to stall infinitely here. I'm not too worried about what that number should be, it could be long if we really need to wait awhile. On my last few upgrades, it didn't seem to take too long. Maybe 5 or 10 minutes? Or is that way to small a time? How long could a potential compute reboot take? jump it to 30 mins?

Reply:
Well, there's a timeout on the UI side that tells the user there's an error when the scripts does not finish in X minutes.

But maybe we could add some timeout also into the script itself...

Reply:
The loop down on line 59 is ok, because if there is an error the status won't be ACTIVE or MIGRATING anymore I suspect. We cloud potentially look at a similar thing if we don't want the timeout approach.